src/AppBundle/Controller/Youtube_videos.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: artur
+ * Date: 11/4/16
+ * Time: 7:05 PM
+ */
+
+namespace AppBundle\Controller;
+
+
+use Doctrine\ORM\Mapping as ORM;
+
+class Youtube_videos
+{
+
+    public function getVideoByKey($planetName)
+    {
+        $url =$this-> getData("https://www.googleapis.com/youtube/v3/search?key=AIzaSyDzxAdrNX8XPi0L4EQQW3kBpUrnHXrbvkM&channelId=UCX6b17PVsYBQ0ip5gyeme-Q&part=id&order=date&maxResults=1&q=".$planetName);
```

Comment:
pirma tai keisti karpai turetu buti `$url = $this->getData(`, antra reiktu padaryti, kad api key pasiimtu is config (tokie dalykai keiciasi ir ju nereikia comitinti i db)